tests/Vagrantfile

```diff
@@ -0,0 +1,52 @@
+# -*- mode: ruby -*-
```

Comment:
You only need a private IP if you are going to assign an hostname to the machine.